cupy/testing/helper.py

```diff
@@ -624,11 +626,14 @@ def _make_all_dtypes(no_float16, no_bool, no_complex):
     if not no_complex:
         dtypes += _complex_dtypes
 
+    if not no_char_code:
+        dtypes += tuple([numpy.dtype(t).char for t in dtypes])
+
     return dtypes
 
 
 def for_all_dtypes(name='dtype', no_float16=False, no_bool=False,
-                   no_complex=False):
+                   no_complex=False, no_char_code=True):
```

Comment:
Ditto. Please separate this change.
We've recently decided to make separate PRs for separate topics as much as possible.

Reply:
Separated these changes to #2152.